test.js

```diff
@@ -1306,6 +1306,29 @@ test('sync error handling', (t) => {
   }
 })
 
+for (const fd of [1, 2]) {
+  test(`fd ${fd}`, (t) => {
+    t.plan(1)
+
+    const fakeFs = Object.create(fs)
+    const SonicBoom = proxyquire('.', {
+      fs: fakeFs
+    })
+
+    const stream = new SonicBoom({ fd })
+
+    fakeFs.close = function (fd, cb) {
+      t.fail('should not close fd 1')
```

Comment:
shouldn't it be `fd` and not hardcoded `1`?

Reply:
Update